packages/compiler/src/experimental/typekit/kits/model.ts

```diff
@@ -32,28 +32,33 @@ interface ModelDescriptor {
 }
 
 export interface ModelKit {
-  model: {
-    /**
-     * Create a model type.
-     *
-     * @param desc The descriptor of the model.
-     */
-    create(desc: ModelDescriptor): Model;
+  /**
+   * Create a model type.
+   *
+   * @param desc The descriptor of the model.
+   */
+  create(desc: ModelDescriptor): Model;
 
-    /**
-     * Check if the given `type` is a model..
-     *
-     * @param type The type to check.
-     */
-    is(type: Type): type is Model;
-  };
+  /**
+   * Check if the given `type` is a model..
+   *
+   * @param type The type to check.
+   */
+  is(type: Type): type is Model;
+}
+
+interface TypeKit {
```

Comment:
This feels inconsitent with all the other type kit.

If we want to change it we should probably change them all(as well as get the doc on how you should name your type kits)

You can reference that model in typescript btw by doing

```
ModelKit["model"] // this type is the type of the model property
```

Reply:
I'm updating it to look like the `[ModelProperty](https://github.com/microsoft/typespec/blob/main/packages/compiler/src/experimental/typekit/kits/model-property.ts)` typekit, since I want to extend from the `Model` kit in the client library type kit I'm adding. So I need to expose a flat interface of the operations. Would a better way to keep it like this, since it's consistent with the `ModelProperty` def, or should I `expose ModelKit["model"] as ModelKit` or something like that, so I can get the flat interface out

Reply:
I guess I am not sure, I would just like our typekit to be consistent(which seems like they already are not if ModelProperty does it that way) @joseharriaga what do you think?

Reply:
also @bterlson, should the template for `ModelProperty` be the required way to define things, so that they would be automatically extensible in other typekits?